src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/RequiredFilesVerifier.java

```diff
@@ -35,6 +35,7 @@ public void verify(HostingRequest request, HashSet<VerificationMessage> hostingI
                 checkSecurityScan(repo, hostingIssues);
                 checkCodeOwners(repo, hostingIssues, forkTo);
                 checkGitignore(repo, hostingIssues);
+                checkDependabot(repo, hostingIssues);
```

Comment:
How about `doCheckDependencyBot` so it could check for either dependabot or renavate or updatecli if it is used in the future?